python/servo/testing_commands.py

```diff
@@ -556,6 +566,37 @@ def filter_intermittents(self, summary, log_filteredsummary, log_intermittents,
                 else:
                     intermittents += [failure]
 
+        if reporter_api:
```

Comment:
No. There's no point until I have an actual URL to give, which requires setting up the service first.